### PR TITLE
Update coexistence-chat-calls-presence.md

### PR DESCRIPTION
**Pull request by luis-msftpm**
Adding details regarding support of scenarios between Multitenant O365 and 21vianet O365 after ICM requesting documentation.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/coexistence-chat-calls-presence.md [hunk 1]
```diff
@@ -168,7 +168,8 @@ Skype for Business threads do not persist beyond the 10 min. SIP session timeout
 Both the in-tenant and federated behaviors described above are available, with the following limitations:
 
 - External attendees whose tenants reside in a different GoLocal deployment or geography won't see IM chat while in a "federated" meeting
-- Federation and interop between Multitenant O365 and Sovereign Clouds is not supported
+- Federation and interop between Multitenant O365 and Office 365 running by 21vianet is supported with limited scenarios.
```
Comment (JohanFreelancer9) [inline Teams/coexistence-chat-calls-presence.md:171]:
```suggestion
- Federation and interop between Multitenant O365 and Office 365 operated by 21Vianet is supported in limited scenarios.
```

File: Teams/coexistence-chat-calls-presence.md [hunk 2]
```diff
@@ -223,6 +224,20 @@ In order to align presence and reachability in pre-existing threads, the target'
 
 In particular, if a recipient you previously had a persistent interop conversation thread with was upgraded to Teams, that thread will no longer reflect accurate presence and will no longer be routable. You should start a new thread.
 
+### Federation and interop with 21vianet
```
Comment (JohanFreelancer9) [inline Teams/coexistence-chat-calls-presence.md:227]:
```suggestion
### Federation and interop with Office 365 operated by 21Vianet
```

File: Teams/coexistence-chat-calls-presence.md [hunk 3]
```diff
@@ -223,6 +224,20 @@ In order to align presence and reachability in pre-existing threads, the target'
...
+
+Federation and interop between Multitenant O365 and 21vianet O365 is supported when the Multitenant O365 users are in TeamsOnly mode. In such a scenario, SFBO users in 21vianet will be able to communicate with TeamsOnly users in Multitenant O365 through chats and calls. The following table shows the supported scenarios in this configuration:
```
Comment (JohanFreelancer9) [inline Teams/coexistence-chat-calls-presence.md:229]:
```suggestion
Federation and interop between Multitenant O365 and Office 365 operated by 21Vianet are supported when the Multitenant O365 users are in TeamsOnly mode. In such a scenario, Skype for Business Online users in Office 365 operated by 21Vianet will be able to communicate with TeamsOnly users in Multitenant O365 through chats and calls. The following table shows the supported scenarios in this configuration:
```

File: Teams/coexistence-chat-calls-presence.md [hunk 4]
```diff
@@ -223,6 +224,20 @@ In order to align presence and reachability in pre-existing threads, the target'
...
+ 
+|Scenario|Origin|Recipient|Supported?|
+|---|---|---|---|
+|Presence|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes<br>Yes|
+|Chat|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 Only)<br>Yes(1:1 Only)|
+|Audio Calls|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 Only)<br>Yes (1:1 Only)|
+|Video Calls|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 Only)<br>Yes (1:1 Only)|
+|Screen Sharing|Teams <br> Skype for Business <br> | Skype for Business <br> Teams |Yes (through a promoted Teams meeting)<br>Yes (through a promoted SfB meeting)|
```
Comment (JohanFreelancer9) [inline Teams/coexistence-chat-calls-presence.md:237]:
```suggestion
|Chat|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 only)<br>Yes(1:1 only)|
|Audio Calls|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 only)<br>Yes (1:1 only)|
|Video Calls|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 only)<br>Yes (1:1 only)|
|Screen Sharing|Teams <br> Skype for Business <br> | Skype for Business <br> Teams |Yes (through a promoted Teams meeting)<br>Yes (through a promoted Skype for Business meeting)|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @luis-msftpm, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**